Updated tooltips to list-like format (#12425)
endless-sky/endless-sky: data/_ui/tooltips.txt

Reviewer: ```suggestion
	`Whether a confirmation dialog will be displayed when you attempt to demand tribute from a planet.`
	`- "off": you will never be asked for confirmation.`
	`- "friendly only": you will only be asked for confirmation against planets that you are friendly toward.`
	`- "always": you will always be asked for confirmation.`
```

	`Toggle vsync between off, on, and adaptive (should your computer support it).`

tip "Camera acceleration"
	`Determines where the screen center is positioned relative to the flagship while accelerating.`
	`- "off": camera is fixed on flagship`
	`- "on": camera trails behind`

> 	`- "reversed": camera speeds ahead`

Reviewer 2: ```suggestion
	`Determines where the screen center is positioned relative to the flagship while accelerating.`
	`- "off": camera is fixed on flagship.`
	`- "on": camera trails behind.`
	`- "reversed": camera speeds ahead.`
```

	`Draw the background starfield with a fixed zoom level instead of changing with the view zoom level.`

tip "Parallax background"

> 	`Add parallax to objects in the background (haze and starfield).`

Reviewer: ```suggestion
	`Add parallax to objects in the background, which includes the haze and starfield.`
```

	`- "fancy": three layers of parallax`
	`- "fast": one layer of parallax`

> 	`- "off": no parallax`

Reviewer: ```suggestion
	`- "fancy": three layers of parallax.`
	`- "fast": one layer of parallax.`
	`- "off": no parallax.`
```

	`Add additional motion blur to the background when jumping between systems. The medium and heavy options control how intense the effect is.`

tip "Ship outlines in shops"
	`Controls the display of your fleet ship icons when in the shipyard or outfitter.`

> 	`- "fancy": applies a sobel filter to all ships (this can result in high GPU load for large fleets).`

Reviewer: ```suggestion
	`- "fancy": applies a sobel filter to all ships. This can result in high GPU load for large fleets and/or on low performance computers.`
```

> 	`- "fancy": applies a sobel filter to all ships. (This can result in high GPU load for large fleets.)`

Reviewer 2: Hmm, not sure here, but maybe this way would be better?

```suggestion
	`- "fancy": applies a sobel filter to all ships (this can result in high GPU load for large fleets).`
```

	`- "fast": just uses the ship's sprite.`

tip "Ship outlines in HUD"
	`Controls the display of your flagship and target ship icons in the HUD.`

> 	`- "fancy": applies a sobel filter to the sprites (this can result in high GPU load for some ships on very old or low performance computers).`

Reviewer: ```suggestion
	`- "fancy": applies a sobel filter to the sprites. This can result in high GPU load for some ships on very old or low performance computers.`
```

> 	`- "fancy": applies a sobel filter to the sprites. (This can result in high GPU load for some ships on very old or low performance computers.)`

Reviewer 2: Same as before, use your own judgement

```suggestion
	`- "fancy": applies a sobel filter to all ships (this can result in high GPU load for large fleets).`
```

Author: I also thought it was weird that "Ship outlines in HUD" and "Ship outlines in shop" have a different parenthesis for their respective "fancy" settings. The difference is that "Ship outlines in HUD" can only affect up to two sprites, your flagship and your target, so it does not matter how large your fleet is, this setting only applies sobel to 1-2 sprites.

I think it should either keep the description it had or simplify entirely to (increases GPU load).

There is also another thing I realized while making sure these settings work like they say they do, the interface _does_ draw your whole escort with a sobel filter (in the bottom left of the screen), but this is not predicated on any setting, it is always the case, which seems weird if applying sobel is a performance concern.

from data/_ui/interfaces.txt:
```
line 1309:
	visible if "!fast hud sprites"
	outline "target sprite"
		center 75 315
		dimensions 70 70
		colored
	visible if "fast hud sprites"
	image "target sprite"
		center 75 315
		dimensions 70 70
...

line 1500:
interface "escort element"
	value "width" 120
	value "basic height" 30
	value "system label height" 15

	anchor top left
	outline "icon"
		center 20 10
		dimensions 20 20
		colored
```
note the use of outline in outline "icon" with no predication compared to the way "target sprite" is handled

Reviewer 2: Oops, I totally missed that the conditions were different on this one...

In that case, my suggestion should have been:
```suggestion
	`- "fancy": applies a sobel filter to the sprites (this can result in high GPU load for some ships on very old or low performance computers).`
```

	`Display an overlay on nearby landable planets and stations.`

tip "Show mini-map"
	`Control the display of a mini-map of nearby systems.`
	`- "when jumping": the mini-map only appears when initiating a hyperspace jump.`
	`- "always on": the mini-map is always present when in flight.`

> 	`- "off": the mini-map is never shown.`

Reviewer: ```suggestion
	`- "off": the mini-map is never shown.`
	`- "when jumping": the mini-map only appears when initiating a hyperspace jump.`
	`- "always on": the mini-map is always present when in flight.`
```

tip "Clickable radar display"
	`Activate the ability to send commands to your escorts using the in-system radar mini-map.`

tip "Alert indicator"

> 	`Indicate when hostile ships enter your current system.`

Reviewer: ```suggestion
	`Control an indicator that occurs when hostile ships enter your current system.`
```

	`- "audio" plays a siren.`

> 	`- "visual" displays an alert icon by the in-system radar.`

Reviewer: ```suggestion
	`- "off": no indicator occurs.`
	`- "audio": plays a siren.`
	`- "visual": displays an alert icon by the in-system radar.`
	`- "both": both audio and visual indicators occur.`
```

	`Automatically fire your primary weapons if your enemy is within the weapon's range.`

tip "Turret tracking"
	`The strategy that your fleet's turrets use when targeting hostiles.`
	`- Focused tracking will target all turrets on a ship at the same hostile.`

> 	`- Opportunistic tracking will cause turrets to target ships that they are nearest to aiming at, allowing multiple hostiles to be engaged at once.`

Reviewer 2: ```suggestion
	`- "focused": all turrets will attack the same target.`
	`- "opportunistic": turrets will target ships that they are nearest to aiming at, allowing multiple hostiles to be engaged at once.`
```

tip "Target asteroid based on"
	`The strategy that the "Select nearest asteroid" control uses for selecting asteroids.`
	`- "proximity": selects the nearest asteroid.`
	`- "value": selects the highest value asteroid.`

tip "Boarding target priority"
	`The strategy that the "Board selected ship" control uses for selecting ships when you have none targeted.`
	`- "proximity": selects the nearest ship.`
	`- "value": selects the highest value ship (with the value being estimated if you haven't scanned the outfits of the ship).`
	`- "mixed": accounts for the value divided by the distance to the ship.`

tip "Escorts expend ammo"
	`The strategy that your escorts use for determining when to fire their secondary weapons. Able to be toggled by the "Fleet: Toggle ammo usage" control.`

> 	`The "frugal" setting only fires secondary weapons when your escort's HP is low or the hostiles in the system are stronger than your fleet and friendlies.`

Reviewer 2: This could maybe also be switched to bullet points? It does work as-is too though.

	`Automatically fire your primary weapons if your enemy is within the weapon's range.`

tip "Turret tracking"
	`The strategy that your fleet's turrets use when targeting hostiles.`
	`- "focused": all turrets will attack the same target.`
	`- "opportunistic": turrets will target ships that they are nearest to aiming at, allowing multiple hostiles to be engaged at once.`

tip "Target asteroid based on"
	`The strategy that the "Select nearest asteroid" control uses for selecting asteroids.`
	`- "proximity": selects the nearest asteroid.`
	`- "value": selects the highest value asteroid.`

tip "Boarding target priority"
	`The strategy that the "Board selected ship" control uses for selecting ships when you have none targeted.`
	`- "proximity": selects the nearest ship.`

> 	`- "value": selects the highest value ship (with the value being estimated if you haven't scanned the outfits of the ship).`

Reviewer: ```suggestion
	`- "value": selects the highest value ship, with the value being estimated if you haven't scanned the outfits of the ship.`
```

	`- "mixed": accounts for the value divided by the distance to the ship.`

tip "Escorts expend ammo"
	`The strategy that your escorts use for determining when to fire their secondary weapons. Able to be toggled by the "Fleet: Toggle ammo usage" control.`

> 	`The "frugal" setting only fires secondary weapons when your escort's HP is low or the hostiles in the system are stronger than your fleet and friendlies.`

Reviewer: ```suggestion
	`- "never": your escorts will never use their secondary weapons.`
	`- "frugal": your escorts will only fire their secondary weapons when their HP is low or the hostiles in the system are stronger than your fleet and friendlies.`
	`- "always": your escorts will always use their secondary weapons when firing.`
```

tip "Flotsam collection"

> 	`Control which of your ships can pick up cargo floating in space.`

Reviewer: ```suggestion
	`Control which of your ships can pick up flotsam (cargo floating in space). Flotsam is picked up automatically when your ships overlap it.`
	`- "off": your ships will never pick up flotsam.`
	`- "flagship only": only your flagship will pick up flotsam.`
	`- "escorts only": only your escorts will pick up flotsam.`
	`- "on": your entire fleet will pick up flotsam.`
```

	`Toggle whether parentheses are shown around potential profits and losses in the ports map mode.`

tip "Notify on destination"

> 	`Indicate when you enter a system with a planet you must land on for a mission.`

Reviewer: ```suggestion
	`Control an indicator that occurs when you enter a system with a planet you must land on for a mission.`
```
